code/modules/clothing/factions/scc.dm

```diff
@@ -35,6 +41,36 @@
 /obj/item/clothing/suit/storage/toggle/armor/vest/scc/toggle_open()
 	return
 
+/obj/item/clothing/suit/storage/toggle/armor/ccia
```

Comment:
Wouldn't it be better to make this into a subset of `suit/storage`? You're not using the functionality of `toggle` at all, and indeed you remove it below.